[Spectrum] Update analytics.md (#14254)
cloudflare/cloudflare-docs: content/spectrum/reference/analytics.md

   -H "X-Auth-Key: 0000" \
   -H "Content-Type: application/json"
```

> {{<content-column>}}

Reviewer: ```suggestion
```

> Review the [Spectrum API documentation](/api/operations/spectrum-aggregate-analytics-get-current-aggregated-analytics) for more example API requests.

Reviewer: ```suggestion
Refer to the [Spectrum API documentation](/api/operations/spectrum-aggregate-analytics-get-current-aggregated-analytics) for more examples of API requests.
```

> {{</content-column>}}

Reviewer: ```suggestion
```
